.github/workflows/windows.yml

```diff
@@ -0,0 +1,38 @@
+name: windows
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - dev
+    paths-ignore:
+      - '**/README.md'
+      - 'LICENSE*'
+      - 'Makefile'
+      - '.github/workflows/linux*'
+      - '.github/workflows/macos*'
+  workflow_dispatch:
```

Comment:
Since all workflow have the same triggers, what about a single workflow with a job per OS? :)

Reply:
that's a good idea, I'll apply the changes in a minute

Reply:
Probably doesn't work right now because @erlef/setup-beam only supports Ubuntu and Windows at the moment

Reply:
We don't have to use the matrix, just have separate job definition for each OS :)

Reply:
I guess this works? Although they are really the same thing for Windows and Linux. Well, we can use a matrix when erlef/setup-beam supports macOS.